modules/apps/knowledge-base/knowledge-base-web/test/admin/js/components/LockedKBArticleModal.js

```diff
@@ -69,5 +114,39 @@ describe('LockedKBArticleModal', () => {
 		});
 
 		expect(getByText('article-in-edition')).toBeInTheDocument();
+		expect(getByText('ok')).toBeInTheDocument();
+	});
+
+	it('renders the modal as group admin when try to move a locked article', async () => {
```

Comment:
This test also covers the previous one (what we want to test is that a modal is opened on the move action - from js- and it is enough to test it as group admin). So I would suggest to delete the previous test `'renders the modal as non group admin when try to move a locked article'.`